base_company_reporting/i18n/ja.po

```diff
@@ -37,3 +37,8 @@ msgstr "レポートに表示する振込先情報です。変更がある場合
 #: model_terms:ir.ui.view,arch_db:base_company_reporting.view_company_form
 msgid "Reporting"
 msgstr "レポーティング"
+
+#. module: base_company_reporting
+#: model:ir.model.fields,field_description:base_company_reporting.field_res_company__contact_team
+msgid "contact_team"
```

Comment:
Adjust this. See the comment on contact_team field definition.

Reply:
Sorry, I didn't understand what you say.
Now I change msgid snake to cammel case.
Tomorrow I will study i18n syntax in Odoo.

Reply:
Thank you, now I sure.